process.pm

```diff
@@ -0,0 +1,311 @@
+# Copyright (C) 2017 SUSE LLC
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation; either version 2 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License along
+# with this program; if not, see <http://www.gnu.org/licenses/>.
+
+package process;
```

Comment:
Seeing as you are using Mojo::Base, shouldn't this be OpenQA::Something::Process?

Reply:
Well, it's not a direct consequence, but sounds somewhat saner to slowly move module namespace's under OpenQA:: (as lowercase modules on first level of namespace are not reccomended, as are reserved to Perl)